### PR TITLE
Enrich the doc so that Marmot looks more friendly to new user.

### PR DESCRIPTION
**Pull request by wongfei2009**

### CODE REVIEW

File: examples/run-cluster.sh [hunk 1]
```diff
@@ -34,11 +34,11 @@ cleanup() {
 
 trap cleanup EXIT
 rm -rf /tmp/nats
-./marmot -config examples/node-1-config.toml -cluster-addr localhost:4221 -cluster-peers 'nats://localhost:4222/' &
+./marmot -config examples/node-1-config.toml -cluster-addr localhost:4221 -cluster-peers 'nats://localhost:4222/,nats://localhost:4223/' &
 job1=$!
 
 sleep 1
-./marmot -config examples/node-2-config.toml -cluster-addr localhost:4222 -cluster-peers 'nats://localhost:4221/' &
+./marmot -config examples/node-2-config.toml -cluster-addr localhost:4222 -cluster-peers 'nats://localhost:4221/,nats://localhost:4223/' &
```
Comment (maxpert) [inline examples/run-cluster.sh:41]: there is no point of passing `:4223` here because that node comes up in very last.

File: docs/src/pages/cli.mdx [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+### Prometheus Configuration
+
+- `bind`: Bind address for Prometheus.
+- `enable`: Enable Prometheus.
+- `namespace`: Namespace for Prometheus.
+- `subsystem`: Subsystem for Prometheus.
+
+
+
+## Sample Config File
```
Comment (maxpert) [inline docs/src/pages/cli.mdx:96]: Not sure if copy pasting a sample config is good idea when there is sample config file that is part of package.

File: go.mod [hunk 3]
```diff
@@ -6,35 +6,38 @@ require (
 	github.com/BurntSushi/toml v1.3.2
 	github.com/asaskevich/EventBus v0.0.0-20200907212545-49d423059eef
 	github.com/denisbrodbeck/machineid v1.0.1
-	github.com/doug-martin/goqu/v9 v9.18.0
+	github.com/doug-martin/goqu/v9 v9.19.0
```
Comment (maxpert) [inline go.mod:9]: Let's make sure this file is excluded, since we are only touching documentation files

File: tests/e2e/marmot_test.go [hunk 4]
```diff
@@ -0,0 +1,63 @@
+package main_test
```
Comment (maxpert) [inline tests/e2e/marmot_test.go:1]: I would have these E2E tests in separate PR. Let's keep this PR focused on docs

File: tests/e2e/marmot_e2e_test.go [hunk 5]
```diff
@@ -0,0 +1,127 @@
+package main_test
```
Comment (maxpert) [inline tests/e2e/marmot_e2e_test.go:1]: Same as below let's keep tests in different PR only go for docs in this PR

**Review by maxpert — CHANGES_REQUESTED** [review]
Couple of changes to be done. Overall I like couple of improvements to doc, but this PR is mixing too many things in single PR. Lets split things up.

File: examples/run-cluster.sh (hunk 1 above)
Comment (wongfei2009) [reply examples/run-cluster.sh:41]: Understand. But I think this optimization makes new user wondering why. Peer configuration symmertic to node 2 and node 3 is more intutive. Also, when you look at the JetStream doc, it's also using a symetric example. 
https://docs.nats.io/running-a-nats-service/configuration/clustering/jetstream_clustering

Indeed, I think specifying two having a different implication in term of HA.

File: docs/src/pages/cli.mdx (hunk 2 above)
Comment (wongfei2009) [reply docs/src/pages/cli.mdx:96]: I think it's much better for the doc, but agree copy & paste is not a good idea. Will try to find if there's way to include inline file in mdx.
